daml.yaml

```diff
@@ -13,12 +13,10 @@ sdk-version: 2.10.0
 # On the other hand, if you use a regular release like:
 # https://github.com/digital-asset/daml/releases/tag/v2.8.0
 # the daml-version would simply be: 2.8.0
-daml-version: 3.3.0-snapshot.20250402.0
+daml-version: 3.3.0-snapshot.20250409.1
 name: daml-ctl
-version: 3.29.0.20250402.1
+version: 3.29.0.20250409.1
```

Comment:
What LF version is being used if you omit it?

Reply:
2.1:

`
daml damlc inspect .daml/dist/daml-ctl-3.29.0.20250409.1.dar | grep daml-lf
daml-lf 2.1

`